nlu/run.py

```diff
@@ -96,7 +96,7 @@
     if os.path.isfile(os.path.join(model.save_dir, "checkpoint")):
         model.restore()
     else:
-        model.save()
+        model.save_model()
```

Comment:
Not sure why we need to rename this.

Reply:
To avoid confusion with keras' built-in save method. It's been throwing aberrant exceptions. Solved it by renaming the boilerplate method